DNN Platform/Admin Modules/Dnn.Modules.ConfigurationManager/XmlMerge.ascx

```diff
@@ -58,9 +58,9 @@
 	jQuery(function($) {
 		$('#<%=cmdSave.ClientID%>, #<%= cmdExecute.ClientID %>').dnnConfirm({
 			text: '<%= ConfirmText %>',
-			yesText: '<%= Localization.GetSafeJSString("Yes.Text", Localization.SharedResourceFile) %>',
-			noText: '<%= Localization.GetSafeJSString("No.Text", Localization.SharedResourceFile) %>',
-			title: '<%= Localization.GetSafeJSString("Confirm.Text", Localization.SharedResourceFile) %>'
+			yesText: '<%= HttpUtility.JavaScriptStringEncode("Yes.Text", Localization.SharedResourceFile) %>',
+			noText: '<%= HttpUtility.JavaScriptStringEncode("No.Text", Localization.SharedResourceFile) %>',
+			title: '<%= HttpUtility.JavaScriptStringEncode("Confirm.Text", Localization.SharedResourceFile) %>'
```

Comment:
Yep, this is an error, I'm recommending we scrap this whole pull request if people are okay with a wholesale swap anyway and use pull #1492